### PR TITLE
Add python script to draw rao-runner load based on the K8S extracted logs of the pods

### PR DESCRIPTION
**Pull request by murgeyseb**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: scripts/extract-rao-runner-load-from-logs.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    else:
+        df = pd.DataFrame(tasks_to_show)
+        fig = px.timeline(df, x_start="Start", x_end="Finish", y="Pod", color="Process")
+        fig.show()
+
+
+def parse_args():
+    arg_parser = argparse.ArgumentParser()
+    arg_parser.add_argument("-i", "--input", help="Kubernetes log file for rao-runner pods", required=True)
+    arg_parser.add_argument("--from-instant", help="Initial date for logs filtering")
```
Comment (schwitzguebel) [inline scripts/extract-rao-runner-load-from-logs.py:68]: when are from_instant and to_instant used ?

Comment (murgeyseb) [reply scripts/extract-rao-runner-load-from-logs.py:68]: Something I forgot to remove .... Not implemented yet

File: scripts/extract-rao-runner-load-from-logs.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+import argparse
+import re
+import plotly.express as px
+import pandas as pd
+import logging
+
+HELP_MESSAGE = "extract-rao-runner-load-from-logs.py -i <logs-file>"
+TIMESTAMP_REGEX = "\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}"
+RAO_REQUEST_RECEIVED = "RAO request received"
+RAO_REQUEST_SENT = "RAO response sent"
```
Comment (kahyami) [inline scripts/extract-rao-runner-load-from-logs.py:11]: I think it could be better to rename this variable RAO_RESPONSE_SENT instead of RAO_REQUEST_SENT

Comment (murgeyseb) [reply scripts/extract-rao-runner-load-from-logs.py]: Done

**Review by kahyami — APPROVED** [review]

**Pull request merged**